File: app/src/main/java/com/example/testingapplication/datamodel/TextView.java
package com.example.testingapplication.datamodel;

import com.fasterxml.jackson.annotation.JsonProperty;

public class TextView {
    @JsonProperty("_android:id")
    public String androidId;
    @JsonProperty("_android:layout_width")
    public String androidLayoutWidth;
    @JsonProperty("_android:layout_height")
    public String androidLayoutHeight;
    @JsonProperty("_android:layout_x")
    public String androidLayoutX;
    @JsonProperty("_android:layout_y")
    public String androidLayoutY;
    @JsonProperty("_android:background")
    public String androidBackground;
    @JsonProperty("_android:letterSpacing")
    public String androidLetterSpacing;
    @JsonProperty("_android:rotation")
    public String androidRotation;
    @JsonProperty("_android:text")
    public String androidText;
    @JsonProperty("_android:textColor")
    public String androidTextColor;
    @JsonProperty("_android:textSize")
    public String androidTextSize;
    @JsonProperty("_android:textAlignment")
    public String androidTextAlignment;
    @JsonProperty("_android:alpha")
    public String androidAlpha;
    @JsonProperty("_android:fontFamily")
    public String androidFontFamily;
}

File: app/src/main/java/com/example/testingapplication/datamodel/TextView.kt
package com.example.testingapplication.datamodel

import com.fasterxml.jackson.annotation.JsonProperty

class TextView {
    @JsonProperty("_android:id")
    var androidId: String? = null

    @JsonProperty("_android:layout_width")
    var androidLayoutWidth: String? = null

    @JsonProperty("_android:layout_height")
    var androidLayoutHeight: String? = null

    @JsonProperty("_android:layout_x")
    var androidLayoutX: String? = null

    @JsonProperty("_android:layout_y")
    var androidLayoutY: String? = null

    @JsonProperty("_android:background")
    var androidBackground: String? = null

    @JsonProperty("_android:letterSpacing")
    var androidLetterSpacing: String? = null

    @JsonProperty("_android:rotation")
    var androidRotation: String? = null

    @JsonProperty("_android:text")
    var androidText: String? = null

    @JsonProperty("_android:textColor")
    var androidTextColor: String? = null

    @JsonProperty("_android:textSize")
    var androidTextSize: String? = null

    @JsonProperty("_android:textAlignment")
    var androidTextAlignment: String? = null

    @JsonProperty("_android:alpha")
    var androidAlpha: String? = null

    @JsonProperty("_android:fontFamily")
    var androidFontFamily: String? = null
}